elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/impl/ElasticsearchIndexManager.java

```diff
@@ -665,7 +665,7 @@ public void performStreamOperation(LuceneWork singleOperation, IndexingMonitor m
 
 	@Override
 	public void optimize() {
-		// TODO Is there such thing for ES?
+		performStreamOperation( OptimizeLuceneWork.INSTANCE, null, false );
```

Comment:
Yes, there is one for ElasticsearchIndexWorkVisitor as we need to implement it. Thought I might as well fix the TODO here as it was a one liner.